.github/workflows/cross-compile.yml

```diff
@@ -45,10 +52,28 @@ jobs:
         run: brew install mingw-w64
         if: startsWith(matrix.image, 'macos-') && matrix.target == 'x86_64-pc-windows-gnu'
 
+      - name: Install LLVM MinGW toolchain
+        run: |
+          name='llvm-mingw-20220323-ucrt-ubuntu-18.04-x86_64'
+          curl -L -o $name.tar.xz https://github.com/mstorsjo/llvm-mingw/releases/download/20220323/$name.tar.xz
+          tar -xf $name.tar.xz
+          echo "$name/bin" >> $GITHUB_PATH
+        if: startsWith(matrix.image, 'ubuntu-') && contains(matrix.target, 'gnullvm') && matrix.version == 'nightly'
```

Comment:
With the number of daily builds, we should be good Internet citizens and utilize a [cache](https://github.com/actions/cache) here.

Something like this should do (untested, probably not functional as-is):

```yaml
env:
  LLVM-MINGW-TOOLCHAIN-NAME: llvm-mingw-20220323-ucrt-ubuntu-18.04-x86_64
# ...
- name: LLVM MinGW toolchain cache configuration
  id: cache-llvm-mingw-toolchain
  uses: actions/cache@v3
  with:
    path: ${ llvm-mingw-toolchain-name } # Assumes .tar.xz extracted in working dir
    key: ${ llvm-mingw-toolchain-name }

- name: Install LLVM MinGW toolchain
  if: startsWith(matrix.image, 'ubuntu-') && contains(matrix.target, 'gnullvm') && matrix.version == 'nightly' && steps.cache-llvm-mingw-toolchain.outputs.cache-hit != 'true'
  run: |
    curl -L -o ${ llvm-mingw-toolchain-name }.tar.xz https://github.com/mstorsjo/llvm-mingw/releases/download/20220323/${ llvm-mingw-toolchain-name }.tar.xz
    tar -xf ${ llvm-mingw-toolchain-name }.tar.xz
    echo "${ llvm-mingw-toolchain-name }/bin" >> $GITHUB_PATH

# ...
```